contracts/controller/SigmaControllerV1.sol

```diff
@@ -71,8 +71,8 @@ contract SigmaControllerV1 is ScoredTokenLists, ControllerConstants {
   // Proxy manager & factory
   IDelegateCallProxyManager public immutable proxyManager;
 
-  // Exit fee recipient for the index pools
-  address public immutable defaultExitFeeRecipient;
+  // Governance address
+  address public immutable governance;
```

Comment:
no corrupt here?

Reply:
Responses left in <https://github.com/indexed-finance/indexed-core/pull/68>